LoopKitUI/Views/Information Screens/InformationScreens.xcassets/glucoseTargetRange mgdL.imageset/Contents.json

```diff
@@ -0,0 +1,26 @@
+{
+  "images" : [
+    {
+      "filename" : "Chart=Correction, Version=V7, Appearance=Light, Unit=mgdL.pdf",
```

Comment:
[nit] I prefer to have the name of the file the same as the name of the asset (with suffixes indicated alternations; e.g. "Dark", "Light", etc.). Less confusion.

Reply:
Ditto for the rest of the new files.

Reply:
I would prefer to keep the filenames exactly as they are exported in Figma (as shown).  That makes it much easier to update.

Reply:
Has design settled on this format strategy for all of the Figma files? What if they change their strategy?

Reply:
I don't think they have a standard strategy for filenames, but there is an "official source" in the Figma file (https://www.figma.com/file/Tf2QUaj26M90tmp4ztS0aY/Therapy-Settings-Acceptance-Flow?node-id=2646%3A851).  As long as I simply export from that source, I should be able to put whatever files here that result from that export, no?